sql/src/test/java/org/apache/druid/sql/calcite/CalciteQueryTest.java

```diff
@@ -8803,6 +8841,7 @@ public void testInnerJoinTwoLookupsToTableUsingNumericColumn() throws Exception
         + "FROM foo\n"
         + "INNER JOIN lookup.lookyloo l1 ON l1.k = foo.m1\n"
         + "INNER JOIN lookup.lookyloo l2 ON l2.k = l1.k",
+        queryContext,
```

Comment:
Should line 8863 use `queryContext` instead of `QUERY_CONTEXT_DEFAULT`?

Reply:
I left it as is to see if the test validates the query context that is used.

It looks like this is done automatically in `BaseCalciteQueryTest#testQuery` ~ line 619